### PR TITLE
Workshop pollen metabarcoding

### PR DESCRIPTION
**Pull request by iimog**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/cctb-it-wuerzburg/installation/pull/19" target="_blank">ReviewNinja

**Comment by greatfireball** [comment]
You are part of the team now, please update your branch to avoid merge conflicts.

File: final_installation/terminal_server/workshop_pollen_metabarcoding/install_packages.R [hunk 1]
```diff
@@ -0,0 +1,5 @@
+install.packages("vegan", repos="https://cran.uni-muenster.de/")
```
Comment (nterhoeven) [inline final_installation/terminal_server/workshop_pollen_metabarcoding/install_packages.R:1]: !fix explain why vegan

Comment (iimog) [inline final_installation/terminal_server/workshop_pollen_metabarcoding/install_packages.R:1]: !fixed vegan is required for basic community data analysis

**Pull request merged**